core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -583,13 +583,13 @@ public T defaultServiceConfig(@Nullable Map<String, ?> serviceConfig) {
   }
 
   /**
-   * Enables or disables service config look-up from the naming system. Enabled by default.
+   * Disables service config look-up from the naming system, which is enabled by default.
    *
    * @return this
    * @since 1.20.0
```

Comment:
since 1.21

Reply:
It will backported to v1.20.x before v1.20.0 is released.